test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -85,8 +85,14 @@ public void testLoadResponseGeneratorModeOfOpNoSuchFactoryMethod() {
     for (Map.Entry<String, String> entry : configEntries.entrySet()) {
       TestHelper.setConfigValue(config, entry.getKey(), entry.getValue());
     }
-    thrown.expect(InvalidConfigurationException.class);
-    DatabaseAdaptor.loadResponseGenerator(config);
+    try {
```

Comment:
Sorry, yes. I thought that would be obvious from the brevity of the JUnit 4 style, but I should have been more clear.